scripts/count_tokens.py

```
@@ -0,0 +1,96 @@
+
+    for filepath in base_path.rglob('*'):
+        if filepath.is_file() and filepath.suffix in args.extensions:
+            tokens = count_tokens_in_file(filepath, encoding)
+            total_tokens += tokens
+            file_count += 1
+            tokens_by_extension[filepath.suffix] += tokens
+            files_by_extension[filepath.suffix] += 1
+
+    # Results
+    print("=" * 60)
+    print(f"📊 RESULTS")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)